be/src/exec/partitioned_aggregation_node.cc

```diff
@@ -849,7 +849,7 @@ Status PartitionedAggregationNode::Partition::Spill(bool more_aggregate_rows) {
     // TODO(ml): enable spill
     std::stringstream msg;
     msg << "New partitioned Aggregation in spill";
-    LIMIT_EXCEEDED(parent->mem_tracker(), parent->state_, msg.str());
+    LIMIT_EXCEEDED(parent->state_->query_mem_tracker(), parent->state_, msg.str());
```

Comment:
I think you can just use `instance_mem_tracker()`
cc @xinyiZzz

Reply:
`instance_mem_tracker()` do not have mem limit so the limit of `instance_mem_tracker`  is -1. `instanc_mem_tracker` is child of `query_mem_tracker()` which have real limit number